### PR TITLE
fix: add error visibiliy on cert parsing path

### PR DESCRIPTION
**Pull request by digikata**
Cert reply parsing can fail w/ a not very expressive error

### CODE REVIEW

File: src/k8-client/src/cert.rs [hunk 1]
```diff
@@ -179,14 +179,23 @@ where
 
             let token_output = Command::new(exec.command.clone())
                 .args(exec.args.clone())
-                .output()?;
+                .output()
+                .map_err(|e| {
+                    let msg = format!("executing {} {:#?}", exec.command, exec.args);
+                    info!(target: "k8_client", "error {msg}");
```
Comment (morenol) [inline src/k8-client/src/cert.rs:185]: shouldn't this be a `debug!`?

Comment (morenol) [reply src/k8-client/src/cert.rs:185]: Oh I see it is an error. Maybe `warn` or `error` instead?

Comment (nacardin) [reply src/k8-client/src/cert.rs:185]: I think it should be `debug!` like the others, since the error is bubbled to the caller.

Comment (digikata) [reply src/k8-client/src/cert.rs:185]: You and Nick had the same comment.  I'll update to `debug!`.  `error!` generates a user visible log line at the default fluvio cli settings that I don't think is desirable.

Comment (sehz) [reply src/k8-client/src/cert.rs:185]: or remove it,  otherwise it will leads to duplicate trace

Comment (nacardin) [reply src/k8-client/src/cert.rs:185]: would it be a duplicate of https://github.com/infinyon/fluvio/blob/ed94fb2e2beb8da70d898c71b329ff197e55afef/crates/fluvio-cluster/src/start/k8.rs#L589 ?

Comment (digikata) [reply src/k8-client/src/cert.rs:185]: Not exactly, without these extra traces, you get an pretty generic io error where you can't differentiate what happened on these paths.  I could remove the traces, and map_err to add more customized context inside a IoError, and then the parent code in the link would work fine.

**Review by nacardin — APPROVED** [review]

**Review by morenol — APPROVED** [review]

**Comment by digikata** [comment]
squashed to single commit

**Pull request merged**